client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -370,6 +378,13 @@ class CohortHeader extends PureComponent {
 
   render() {
     const { pendingForDescription } = this.state;
+    const {
+      details: { locks }
+    } = this.props;
+
+    if (!locks) {
```

Comment:
Please assign the default of `{ name: true, description: true }` and remove the conditionals - assume the item is locked until loaded.